bots/images/scripts/fedora.setup

```diff
@@ -157,7 +155,7 @@ srpm=$(/var/lib/testvm/make-srpm $TEST_SOURCE)
 #
 # https://bugzilla.redhat.com/show_bug.cgi?id=1447627
 
-su builder -c "/usr/bin/mock --no-bootstrap-chroot --verbose --installdeps \"$PWD/$srpm\""
+su builder -c "/usr/bin/mock --no-bootstrap-chroot --verbose -i $(curl https://raw.githubusercontent.com/cockpit-project/cockpit/master/tools/cockpit.spec | sed -n '/^BuildRequires:/ {s/^.*: //; s/>=.*$//; s/%{.*}//; s/^[ ]*//; s/[ ]*$//; s/\(.*\)/\"\1\"/; s/ /" "/; p}' | tr '\n' ' ')"
```

Comment:
Let's please split that into two or more lines. This could also use a comment what the fairly involved seddery does. Does it really need to be that involved? My [own script](https://piware.de/tools/bin/mockspec) for this is a lot simpler, and has worked fine so far?

Perhaps it makes sense to split that out into a bots/images/scripts/lib/ script,  so that it can be reused between fedora.setup and rhel.setup?

Reply:
I'll comment it a bit better.

So I started from your script, but it errors with '(' or ')' in title, so it needs a quote around each package. Now I think it would be better to make this so it only puts quotes around the one with brackets. Because I ran into other issues which is why it turned out so complicated. (Multiple packages per BuildRequires line, so the quote was misplaced, random trailing spaces etc...)